5e-SRD-Monsters.json

```diff
@@ -122,8 +122,12 @@
         ]
       },
       {
-        "name": "Enslave (3/day)",
+        "name": "Enslave",
```

Comment:
Should we drop `(3/day)` from the name? That is the full name of the ability in SRD and source material. It also keeps it  human readable. Even if it does feel silly to keep  it in there.

Reply:
No strong opinion, you have the object split out, which is nice.  I guess if you wanted to query across lots of things for the "enslave",its easier for exact match, in case others would have `(2/day)` or different values. Just thinking out loud, could do a substring search to avoid that, but might want it simple.

Reply:
@bagelbits no, it's not part of the name. Just because it has the same style doesn't mean it is. The SRD list those as limited usage "notations" (see page 259). It's the same case for "costs 2 (or 3) actions" text. It's not part of the name: it's only the same style as a mnemonic for quick reference.